src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -152,6 +152,28 @@ public function compileDropIfExists(Blueprint $blueprint, Fluent $command)
     }
 
     /**
+     * Compile a drop all tables command.
+     *
+     * @param  string  $tables
+     * @return string
+     */
+    public function compileDropAllTables($tables)
+    {
+        return 'drop table '.implode(',', $tables).' cascade';
```

Comment:
Shouldn't some sort of escaping be used here? It's rarely, but schema and tables may have really weird names